src/ray/core_worker/actor_manager.h

```diff
@@ -22,6 +22,46 @@
 
 namespace ray {
 
+class ActorCreatorInterface {
+ public:
+  virtual ~ActorCreatorInterface() = default;
+  /// Register actor to GCS asynchronously.
+  ///
+  /// \param task_spec The specification for the actor creation task.
+  /// \param callback Callback that will be called after the actor info is written to GCS.
+  /// \return Status
+  virtual Status AsyncRegisterActor(const TaskSpecification &task_spec,
```

Comment:
This is not async?

Reply:
> This is not async?

Thank you for your advice. Interfaces should really be defined as synchronous to make the semantics more clearer.